packages/affiliates/gas-rebate/VoterGasRebate.js

```diff
@@ -35,6 +35,16 @@ const argv = require("minimist")(process.argv.slice(), {
 const fs = require("fs");
 const path = require("path");
 
+// Locally stored list of DesignatedVoting wallets to substitute with their hot wallets:
+let addressesToReplace = {};
+try {
+  if (fs.existsSync("./2KEY_ADDRESS_OVERRIDE.json")) {
+    addressesToReplace = require("./2KEY_ADDRESS_OVERRIDE.json");
+  }
+} catch (err) {
+  // Do nothing if file is not provided.
+}
```

Comment:
Why would this error if you're checking that the file exists?

Reply:
Good point, removed